### DISCUSSION TITLE
Upgrade from Big Sur

### DISCUSSION THREAD

**Discussion by joachimd90** [valnoxy/t480-oc#67] [General]
I have a freshly installed Big Sur on my T480s with the EFI but looking into upgrade to Venture.
Would this need a fresh install or could I use MacOS upgrade thru Software Upgrade?

**Comment by valnoxy** [comment]
You can upgrade via Software Upgrade without any problems. However, you need the EFI configuration version 1.2.2 or higher, so that the correct kexts are loaded under Ventura.
